### PR TITLE
fix return statement without argument crash

### PR DESCRIPTION
**Pull request by connectdotz**
fix parsing error for  no argument return statement: `return;`

### CODE REVIEW

**Comment by connectdotz** [comment]
@firsttris do you have a few minutes to review this?

File: src/parsers/babel_parser.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ export const parse = (file: string, data: ?string, options: ?parser.ParserOption
         isFunctionDeclaration(element.expression.right.type)
       ) {
         searchNodes(element.expression.right.body, parent);
-      } else if (element.type === 'ReturnStatement' && element.argument.arguments) {
+      } else if (element.type === 'ReturnStatement' && element.argument && element.argument.arguments) {
```
Comment (firsttris) [inline src/parsers/babel_parser.js]: `else if (element.type === 'ReturnStatement' && element.argument?.arguments)`

you could use optional chaining here instead.
even if its still javascript it's already used a few times in this file.

**Review by firsttris — COMMENTED** [review]
of course it's a pleasure

File: src/parsers/babel_parser.js (hunk 1 above)
Comment (connectdotz) [reply src/parsers/babel_parser.js]: good point

**Comment by connectdotz** [comment]
regarding coverage decrease: looking at the detailed report, the `src` coverage has not changed but the overall coverage reduced due to the removal of `fixures`, which should not have been included anyway.

**Pull request merged**